src/main/java/edu/whimc/sciencetools/models/sciencetool/NumericScienceTool.java

```diff
@@ -64,23 +65,31 @@ public NumericScienceTool(String toolKey,
     @Override
     public @Nullable String displayMeasurement(Player player) {
         // check if player in disabled world
+
         if (super.disabledWorlds.contains(player.getWorld())) {
-            Utils.msg(player, "&cWe don't know how to measure that here!");
+            if(messages.containsKey("disabled-in-world"))
+                Utils.msg(player, messages.get("disabled-in-world").toString());
+            else
+                Utils.msg(player, ScienceToolManager.getGlobalMassages().get("disabled-in-world").toString());
             return null;
         }
 
         double data = getData(player.getLocation());
+        String message;
+        if(messages.containsKey("numeric-measure-format"))
+            message = messages.get("numeric-measure-format").displayString(displayName,Utils.trimDecimals(data, precision),unit);
+        else {
 
-        StringBuilder message = new StringBuilder("&aThe measured " + this.displayName
-                + " is &f" + Utils.trimDecimals(data, this.precision) + this.unit + "&7");
-
+            message = ScienceToolManager.getGlobalMassages().get("numeric-measure-format").displayString(displayName, Utils.trimDecimals(data, precision), unit);
+        }
         // display converted values
-        for (Conversion conversion : this.conversions) {
-            String converted = Utils.trimDecimals(conversion.convert(data), this.precision);
-            message.append(" (").append(converted).append(conversion.getUnit()).append(")");
+        for (Conversion conversion : conversions) {
+            String converted = Utils.trimDecimals(conversion.convert(data), precision);
+            message+=" ("+converted+conversion.getUnit()+")";
```

Comment:
```suggestion
            message += " (" + converted + conversion.getUnit() + ")";
```
Formatting nitpick again sorry